lua/entities/gmod_tardis/modules/cl_music.lua

```diff
@@ -42,21 +42,21 @@ function ENT:ResolveMusicURL(url)
 					local tbl=util.JSONToTable(body)
 					if tbl then
 						if tbl.success then
-							LocalPlayer():ChatPrint("Playing: "..tbl.title)
+                            TARDIS:Message(LocalPlayer(), "Playing: " ..tbl.title)
 							self:PlayMusic(api.."play?id="..id,true)
 						else
-							LocalPlayer():ChatPrint("ERROR: Failed to load ("..(tbl.err and tbl.err or "Unknown reason")..")")
+                            TARDIS:Message(LocalPlayer(), "ERROR: Failed to load ("..(tbl.err and tbl.err or "Unknown reason")..")")
 						end
 					else
-						LocalPlayer():ChatPrint("ERROR: Failed to load API response")
+                        TARDIS:Message(LocalPlayer(), "ERROR: Failed to load API response")
 					end
 				end,
 				function(err)
-					LocalPlayer():ChatPrint("ERROR: Failed to resolve url ("..err..")")
+                    TARDIS:Message(LocalPlayer(), "ERROR: Failed to resolve url ("..err..")")
 				end
 			)
 		else
-			LocalPlayer():ChatPrint("ERROR: Couldn't find video ID inside url")
+            TARDIS:Message(LocalPlayer(), "ERROR: Couldn't find video ID inside url")
```

Comment:
Also, it's better to use TABs than spaces, since that's what the project currently has. 
As you can see, the github GUI has all the offsets messed up